packages/suite/src/hooks/wallet/useSendFormChangeHandlers.ts

```diff
@@ -0,0 +1,54 @@
+import { SendContextValues } from 'src/types/wallet/sendForm';
+
+export type HandleChangeParams = {
+    outputId: number;
+    value: string;
+};
+
+type HandleAmountOrFiatChangeParams = HandleChangeParams & {
+    calculateTheOtherValue: CalculateTheOtherValue;
+};
+
+type CalculateTheOtherValue = (outputId: number, value: string) => void;
+
+type UseSendFormChangeHandlersParams = {
+    calculateAmountFromFiat: CalculateTheOtherValue;
+    calculateFiatFromAmount: CalculateTheOtherValue;
+    composeRequest: SendContextValues['composeTransaction'];
+    setValue: SendContextValues['setValue'];
+};
+
+export const useSendFormChangeHandlers = ({
+    calculateAmountFromFiat,
+    calculateFiatFromAmount,
+    composeRequest,
+    setValue,
+}: UseSendFormChangeHandlersParams) => {
+    const handleAmountOrFiatChange = ({
```

Comment:
I moved the change handlers here because there was duplicate code in `Amount` and `FiatInput`. This is certainly more lines, but avoiding repetition is worth it in my opinion.

Reply:
That sounds good to centralize change handlers here, but are you sure about the `handleAmountOrFiatChange` and `calculateTheOtherValue`?
This layer of abstraction add complexity, I think it'd just fine if there were too functions `handleAmountChange` and `handleFiatChange` with some boilerplate. But ok :shrug:

Reply:
I agree that was a bit wild. What about 7200dfcd4a8970a766a1ec936c719f515a3d8cac?